distribution/docs/src/main/resources/content/_metadataAttributes/attribute-mappings.adoc

```diff
@@ -1,5 +1,5 @@
 :title: Catalog Taxonomy Attribute Mappings
-:type: appendix
+:type: metadataReference
```

Comment:
❓ Should this be `subMetadataReference`?

Reply:
It's been a sibling to Catalog Taxonomy Definitions. Making it a `subMetadataReference` would make it a subsection instead of a sibling